recipes/cwl-utils/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cwl-utils" %}
+{% set version = "0.18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cwl-utils-{{ version }}.tar.gz
+  sha256: 43e57318db207310e46726df1081c55ceddebab888e09484d1175f4c9a103384
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - cwl-upgrader >=1.2
+    # From extras[pretty], for cwl-graph-split. An alternative for this would be to build for
+    # multiple outputs, e.g. https://github.com/conda-forge/matplotlib-feedstock/blob/2ca6433797f0a4f77fee890c6fa39617a6a6b0b4/recipe/meta.yaml#L17
+    - cwlformat
+    - packaging
+    - python >=3.6
+    - requests
+    - schema-salad >=8.2,<9
```

Comment:
These package specs do not match upstream.

https://github.com/common-workflow-language/cwl-utils/blob/v0.18/requirements.txt

Reply:
Done. Now the only difference should be the python>3.6 and the cwlformat (it's an extra in the upstream https://github.com/common-workflow-language/cwl-utils/blob/v0.18/setup.py#L42).